### PR TITLE
{mpi}[GCC/7.2.0-2.29] MPICH v3.2

### PR DESCRIPTION
**Pull request by rubendibattista**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name = 'MPICH'
+version = '3.2'
```
Comment (boegel) [inline easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb]: @rubendibattista Any reason you're not updating to `3.2.1`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2539.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/da60cf7fd5c4fed6e83f7db861cd05b7 for a full test report.

File: easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb (hunk 1 above)
Comment (rubendibattista) [reply easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb]: Not really. I just edited the one that was there with a previous GCC. 

I'm planning to PR `gmpich-2018a` with GCC-7.3 and MPICH 3.2.1. How does it work with toolchains? Should I do something special or wait?

Comment (boegel) [reply easybuild/easyconfigs/m/MPICH/MPICH-3.2-GCC-7.2.0-2.29.eb]: It usually makes sense to check whether there are any software updates, bug fixes can always be useful.
I see you've issued #5883 for MPICH 3.2.1, we can merge both, that's fine.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d7a859e2216fe12154a8ae8cbb3413ca for a full test report.

**Comment by boegel** [comment]
Going in, thanks @rubendibattista!

**Pull request merged**